Update the readme (#261)
infinum/eightshift-boilerplate: README.md

[![GitHub stars](https://img.shields.io/github/stars/infinum/eightshift-boilerplate.svg?style=for-the-badge&label=Stars)](https://github.com/infinum/eightshift-boilerplate/)
[![license](https://img.shields.io/github/license/infinum/eightshift-boilerplate.svg?style=for-the-badge)](https://github.com/infinum/eightshift-boilerplate)

## About Eightshift Boilerplate

> Eightshift Boilerplate is a modern WordPress theme boilerplate. It includes everything you need to start building a modern WordPress project, using modern front end development tools, and it's at the core of the Eightshift Development Kit. [Find out more in Eightshift Docs &rarr;](https://infinum.github.io/eightshift-docs/)

Reviewer: ```suggestion
Eightshift Boilerplate is a modern WordPress theme boilerplate. It includes everything you need to start building a modern WordPress project, using state-of-the-art front-end development tools, and it's at the core of the Eightshift Development Kit. [Find out more in Eightshift Docs &rarr;](https://infinum.github.io/eightshift-docs/)
```

Or similar wording because we already have `modern WordPress` in the same sentence.

> Eightshift Boilerplate is a modern WordPress theme boilerplate. It includes everything you need to start building a modern WordPress project, using modern front end development tools, and it's at the core of the Eightshift Development Kit. [Find out more in Eightshift Docs &rarr;](https://infinum.github.io/eightshift-docs/)

Author: Eightshift Boilerplate is a modern, Gutenberg-oriented, WordPress theme boilerplate or something like that?

> Eightshift Boilerplate is a modern WordPress theme boilerplate. It includes everything you need to start building a modern WordPress project, using modern front end development tools, and it's at the core of the Eightshift Development Kit. [Find out more in Eightshift Docs &rarr;](https://infinum.github.io/eightshift-docs/)

Reviewer 2: Sure, sounds better

## ⚠️ Requirements
To get started, you'll need:

> * [npm](https://www.npmjs.com/)

Reviewer 3: Should this be replaced with Node LTS?

> * [composer](https://getcomposer.org/)

Reviewer 3: ```suggestion
* [Composer](https://getcomposer.org/)
```

* [git](https://git-scm.com/)
* [WP-CLI](https://wp-cli.org/)

## 🏁 Quick start

If you'd like to get started ASAP, run `npx create-wp-project` inside of `wp-content/themes` to bootstrap your Eightshift Development Kit project.

[Documentation on getting started with Eightshift Boilerplate is available in Eightshift Docs &rarr;](https://infinum.github.io/eightshift-docs/docs/theme/)

## 📚 Documentation

> Eightshift Development Kit comes with [comprehensive documentation](https://infinum.github.io/eightshift-docs).

Reviewer 3: ```suggestion
Eightshift Development Kit comes with a [comprehensive documentation](https://infinum.github.io/eightshift-docs).
```

Author: "Documentation" is very rarely used as a countable noun (see https://en.wiktionary.org/wiki/documentation#Etymology_1), and we don't use it as a countable noun anywhere on the site?

## 💻 Other resources

* [Documentation](https://infinum.github.io/eightshift-docs/)
* [Tutorials and more on the blog](https://infinum.github.io/eightshift-docs/blog)
* [Storybook](https://infinum.github.io/eightshift-docs/storybook/)

> * [Sassdocs](https://infinum.github.io/eightshift-docs/showcase)

Author: It should have been `SassDoc` 😄  http://sassdoc.com/

> * [Sassdocs](https://infinum.github.io/eightshift-docs/showcase)

Reviewer 2: ```suggestion
* [SassDoc](http://sassdoc.com)
```

* Other parts of the Development kit:
* * [Eightshift Boilerplate for Plugins](https://github.com/infinum/eightshift-boilerplate-plugin)
* * [Eightshift Frontend Libs](https://github.com/infinum/eightshift-frontend-libs)
* * [Eightshift Libs](https://github.com/infinum/eightshift-libs)

## 🛟 Getting help

If you have any questions or problems, please [open an issue](https://github.com/infinum/eightshift-boilerplate/issues) on GitHub. 

> Please follow our [code of conduct](https://github.com/infinum/eightshift-boilerplate/blob/develop/CODE_OF_CONDUCT.md), and fill out the issue template properly. We'll do our best to answer your issues as quickly as humanly possible: following these steps helps us out a lot in doing that.

Author: ```suggestion
When submitting issues or otherwise participating in development, follow our [code of conduct](https://github.com/infinum/eightshift-boilerplate/blob/develop/CODE_OF_CONDUCT.md), and fill out the issue template properly. We'll do our best to answer your issues as quickly as humanly possible: following these steps helps us out a lot in doing that.
```

> Please follow our [code of conduct](https://github.com/infinum/eightshift-boilerplate/blob/develop/CODE_OF_CONDUCT.md), and fill out the issue template properly. We'll do our best to answer your issues as quickly as humanly possible: following these steps helps us out a lot in doing that.

Reviewer 2: Nice, much better indeed
